tests/test_python.py

```diff
@@ -100,11 +101,19 @@ def test_pip(auto_container):
     assert f"pip {version_from_env}" in reported_version
 
 
+@pytest.mark.skipif(
+    OS_VERSION == "tumbleweed",
+    reason="pip --user not working properly with PEP 668",
```

Comment:
This sounds a bit better maybe?
```suggestion
    reason="pip --user not working properly due to PEP 668",
```